src/index.js

```diff
@@ -19,10 +19,20 @@ function init( {
 		showPopup = ( { target } ) => {
 			const title = target.getAttribute( 'data-wp-title' ) || target.textContent,
 				lang = target.getAttribute( 'data-wp-lang' ) || globalLang
+
 			requestPagePreview( lang, title, isTouch, data => {
 				if ( data ) {
 					popup.show( renderPreview( lang, data, isTouch ), target )
 				}
+
+				popup.lang = lang
+				popup.title = title
+				const expanded = document.querySelector( '.wikipediapreview.expanded.mobile' )
+				if ( expanded ) {
+					requestPageMedia( lang, title, mediaData => {
+						renderPreviewMedia( document, mediaData )
+					} )
```

Comment:
moving this into the event or util, so we can reuse in both index.js and event.js